mentat/code_context.py

```diff
@@ -72,26 +74,18 @@ def display_context(self):
         stream = session_context.stream
         config = session_context.config
         git_root = session_context.git_root
+        prefix = "  "
 
+        stream.send("")
         stream.send("Code Context:", color="blue")
```

Comment:
I notice we use three different shades of blue: blue, light_blue and cyan. I wonder if we should consolidate?
![image](https://github.com/AbanteAI/mentat/assets/385534/f5ac5099-8c1d-4dc3-b129-13a650ba5510)

Reply:
I tried to update it so that cyan = context/cost, and light_blue = instructions/edits. Here's a full convo for reference.
<img width="632" alt="image" src="https://github.com/AbanteAI/mentat/assets/50287275/956dc2cc-956f-4cdd-a9a6-5830e3f65fab">